bundle/jsky.app.ot.testlauncher/src/main/scala/jsky/app/ot/testlauncher/ServiceChecker.scala

```diff
@@ -0,0 +1,42 @@
+package jsky.app.ot.testlauncher
+
+import edu.gemini.sp.vcs2.VcsService
+
+import scalaz._
+import Scalaz._
+import edu.gemini.util.trpc.auth.TrpcKeyChain
+import edu.gemini.util.security.auth.keychain.Action._
+import java.io.File
+
+import edu.gemini.auxfile.server.AuxFileServer
+import edu.gemini.itc.shared.ItcService
+import edu.gemini.pot.spdb.IDBQueryRunner
+import edu.gemini.services.client.TelescopeScheduleService
+import edu.gemini.spModel.core.{Peer, Site}
+import edu.gemini.too.event.api.TooService
+import edu.gemini.util.security.auth.keychain.KeyService
+import edu.gemini.util.trpc.client.TrpcClient
+
+object ServiceChecker extends App {
```

Comment:
This is just a sanity check that you can run from IDEA to make sure services are responding.